src/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
-    "name": "GLIB-ChromeExt",
-    "version": "1.5",
+  "name": "GLIB-ChromeExt",
+  //"key": "glib", Used for local dev
+  "version": "1.5",
     "description": "GLIB-ChromeExt",
     "options_page": "options.html",
     "permissions": [
```

Comment:
So one important thing to note.  There is a EXTENSION_ID value in config.js that is used for setting the callback URL on auth calls to TC.  You will need to update that value to match the production app value prior to pushing the update to the Chrome Web Store.  There are ways to dynamically detect this value and we should update to do this eventually.  I'll open a ticket.